articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,87 @@
+When it comes to web development for non-techies the first thought that pops into their heads is a deep dive into a dark abyss. Where you magically open your _third eye_ and manage to talk to the computer in a language only the chosen few can understand. However, this is far from reality. Change sides and take it from a techies perspective. The first thoughts that pop up might be "What framework best fits this task?" or "Should I do this from scratch or customize an existing template?". A great number of us look forward to starting a new project. This to us means learning a different concept or re-enforcing an existing one. But what if you decide to build a new webpage from scratch? Do you remember the basics? How are you going to implement your CSS on the webpage? CSS is part of the backbone of any webpage but do you know that it also has its form of styling? Yes we all know about bootstrap and its ease of use but that is a topic for a different day.
+
+### Styling in CSS
+A simple google search will show you that there three different styling methods in CSS each different from the other. Each level has its application and use scenario depending on what gets the job done.
+
+  1. **Local styles:** Also known as inline. This form is defined within your HTML tags/elements. It is mostly used to style specific elements in your code.
+  ~~~HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+    </head>
+    <body>
+      <p style = "font-family: sans-serif;
+          font-size: 1.2em
+          font-style: italic;">
+          This paragraph is an example of a local style.
+        </p>
+    </body>
+  </html>
+  ~~~
+
+  2. **Page-Level styles:** Page-level styles are defined at the header area of the HTML file. Unlike local styles, you can customize all the similar tags at once thereby saving time and maintaining uniformity.
+  ~~~HTML
+  <html>
+    <head>
+      <title>Cascading Style Sheets</title>
+      <meta charset="utf-8">
+      <style type="text/css">
+         body{
+           color: yellow;
+           background-color: red;
+         }
+         p{
+           color: red;
+           background-color: yellow;
+         }
+      </style>
+    </head>
+    <body>
+      <h1>Heading</h1>
+      <p>This paragraph has been styled using page level styling.</p>
+    </body>
+  </html>
+  ~~~
+
+  3. **External Styles** In this case, the styles used for the webpage are located in a completely different file. They are linked to the HTML file depending on which CSS file (Style) best fits the webpage. When developing a website with multiple pages this styling usually comes in handy.
```

Comment:
Remove "In this case" to be parallel to first 2 items in list.

"They are linked to the HTML file depending on which CSS file (Style) best fits the webpage." => This is confusing. This bullet point should just be saying that the HTML file is linked to a .css file which can be imported to modify the style.

"When developing a website with multiple pages this styling usually comes in handy." => Since you don't make any reference to the use case for the first two, you can move this to the pros/cons of each in the next section.